src/Tribe/Commerce/PayPal/Gateway.php

```diff
@@ -113,7 +116,12 @@ public function add_to_cart() {
 		$cart_url      = $this->get_cart_url( '_cart' );
 		$post_url      = get_permalink( $post );
 		$currency_code = trim( tribe_get_option( 'ticket-commerce-currency-code' ) );
-		$product_ids   = (array) $_POST['product_id'];
+
+		if ( isset( $_POST['tribe_tickets'] ) ) {
+			$product_ids = wp_list_pluck( $_POST['tribe_tickets'], 'ticket_id' );
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['tribe_tickets']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: fc6c56142bf65c9a7700e963021e0c4e</sub>